src/software/physics/physics.h

```diff
@@ -36,3 +38,152 @@ Point calculateFuturePosition(const Point &initial_position,
 Vector calculateFutureVelocity(const Vector &intial_velocity,
                                const Vector &constant_acceleration,
                                const Duration &duration_in_future);
+
+
+/**
+ * Vector representation of 2D Euclidean space.
+ *
+ * 0: X
+ * 1: Y
+ * 2: W/Rotation
+ */
+typedef Eigen::Vector3d EuclideanSpace_t;
+
+/**
+ * Vector representation of the robot wheel space.
+ *
+ * 0: Front Right
+ * 1: Back Right
+ * 2: Back Left
+ * 3: Front Left
+ * TODO: Verify wheel orientations
```

Comment:
lets pick the indexing thats consistent with the omindrive code
ill update the motor code to reflect that as well